docs/reference/vectors/vector-functions.asciidoc

```diff
@@ -195,3 +205,51 @@ You can check if a document has a value for the field `my_vector` by
 "source": "doc['my_vector'].size() == 0 ? 0 : cosineSimilarity(params.queryVector, 'my_vector')"
 --------------------------------------------------
 // NOTCONSOLE
+
+The recommended way to access dense vectors is through `cosineSimilarity`,
+`dotProduct`, `l1norm` or `l2norm` functions. But for custom use cases,
+you can access dense vectors's values directly through the following functions:
+
+- `doc[<field>].vectorValue` – returns a vector's value as an array of floats
+
+- `doc[<field>].magnitude` – returns a vector's magnitude as a float
+(for vectors created prior version 7.5 magnitude is not stored.
```

Comment:
Small corrections: "for vectors created prior **to** version 7.5 **the** magnitude is not stored. So this function calculates it anew every time **it** is called"

Reply:
Addressed in a66f75590ba961439ac02907a46074312d3a80a2